modules/system/classes/PluginManager.php

```diff
@@ -680,11 +680,13 @@ public function registerReplacedPlugins()
             if (!isset($this->plugins[$target])) {
                 // register lang namespace alias for bc
                 Lang::registerNamespaceAlias($replacement, $target);
+                Config::getLoader()->registerNamespaceAlias($replacement, $target);
                 continue;
             }
 
             if ($this->plugins[$replacement]->replaces($target, $this->plugins[$target]->getVersion())) {
                 Lang::registerNamespaceAlias($replacement, $target);
+                Config::getLoader()->registerNamespaceAlias($replacement, $target);
```

Comment:
@jaxwilko is there a reason why we want to have config namespace aliasing registered in two separate locations?

Reply:
I'm more so talking about the two different registerNamespaceAlias methods in https://github.com/wintercms/storm/pull/19